dts/bindings/regulator/cirrus,cp9314.yaml

```
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 Cirrus Logic, Inc.
+# SPDX -License-Identifier: Apache-2.0
+
+description: Cirrus CP9314 Buck Switched Cap DC/DC Converter
+
+compatible: "cirrus,cp9314"
+
+include:
+  - name: i2c-device.yaml
+  - name: regulator.yaml
+
+properties:
+  reg:
+    required: true
+
+  cirrus,en-gpios:
+    type: phandle-array
+    description: GPIO tied to EN pin
+
+  cirrus,initial-converter-mode:
+    type: int
+    required: true
```

As suggested by Gerard, you can use enums here, e.g:

```suggestion
    type: string
    required: true
    enum:
    - "2:1"
    - "3:1"
```

---

ack, thanks